src/sas/qtgui/Utilities/WhatsNew/WhatsNew.py

```diff
@@ -145,32 +147,58 @@ def __init__(self, parent=None, strictly_newer=True):
 
         # Callbacks
         self.closeButton.clicked.connect(self.close_me)
+        self.prevButton.clicked.connect(self.prev_file)
         self.nextButton.clicked.connect(self.next_file)
 
         # # Gather new files
-        new_messages = whats_new_messages(strictly_newer=strictly_newer)
+        new_messages = whats_new_messages(only_recent=only_recent)
         new_message_directories = [key for key in new_messages.keys()]
-        new_message_directories.sort(key=reduced_version)
+        new_message_directories.sort(key=reduced_version, reverse=True)
 
         self.all_messages = []
 
-        for version in new_messages:
+        for version in new_message_directories:
             self.all_messages += new_messages[version]
 
         self.max_index = len(self.all_messages)
         self.current_index = 0
 
         self.show_file()
+        self.set_enable_disable_prev_next()
 
         self.setFixedSize(800, 600)
         self.setModal(True)
 
     def next_file(self):
+        """ Show the next available file (increment counter, show)"""
         self.current_index += 1
         self.current_index %= self.max_index
         self.show_file()
+        self.set_enable_disable_prev_next()
+
+    def prev_file(self):
+        """ Go to next file"""
+        self.current_index -= 1
+        if self.current_index < 0:
+            self.current_index = self.max_index - 1
+        self.show_file()
+        self.set_enable_disable_prev_next()
+
+    def set_enable_disable_prev_next(self):
+        """ Set the appropriate enable state on the navigation buttons"""
+
+        if self.current_index == 0:
+            self.prevButton.setEnabled(False)
+        else:
+            self.prevButton.setEnabled(True)
+
+        if self.current_index >= self.max_index - 1:
+            self.nextButton.setEnabled(False)
+        else:
+            self.nextButton.setEnabled(True)
```

Comment:
or just
```
def set_enable_disable_prev_next(self):
    """ Set the appropriate enable state on the navigation buttons """
    self.prevButton.setEnabled(self.current_index > 0)
    self.nextButton.setEnabled(self.current_index < self.max_index - 1)
```

Reply:
@jamescrake-merani - see!
@rozyczko - I removed the comment above this that said "this is good code style" (just my opinion on it: it is much easier to follow like this)